Dockerfile.ocp

```
@@ -0,0 +1,52 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_1.22 AS build
+
+#ENV GOPATH=/go
+#ENV PATH=$GOPATH/bin:/usr/local/go/bin:$PATH
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
```

can you update this to upload the entire repo? This will use the vendored modules.

```suggestion
COPY . .
```